### PR TITLE
1030 take care of lic grants

### PR DESCRIPTION
**Pull request by eparovyshnaya**

### CODE REVIEW

File: bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java [hunk 1]
```diff
@@ -62,7 +62,8 @@ public void restart() {
 
 	public void state() {
 		try {
-			System.out.println(server.state());
+			String where = port.get().map(i -> i.toString()).orElse("-"); //$NON-NLS-1$
+			System.out.println(server.state() + " on port " + where); //$NON-NLS-1$
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java:66]: System.out?

File: bundles/org.eclipse.passage.lbc.jetty/config/log4j2.xml [hunk 2]
```diff
@@ -40,11 +32,13 @@
 
 	<Loggers>
 
-		<Root level="debug">
+		<Root level="error">
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lbc.jetty/config/log4j2.xml:35]: excellent!

File: bundles/org.eclipse.passage.lbc.jetty/config/log4j2.xml [hunk 3]
```diff
@@ -1,16 +1,8 @@
...
-	This program and the accompanying materials are made available under the
-	terms of the Eclipse Public License 2.0 which is available at
-	https://www.eclipse.org/legal/epl-2.0/.
-
-	SPDX-License-Identifier: EPL-2.0
-
-	Contributors:
-		ArSysOp - initial API and implementation
--->
+<!-- Copyright (c) 2021 ArSysOp This program and the accompanying materials
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.lbc.jetty/config/log4j2.xml:2]: -1
please revert

**Review by ruspl-afed — CHANGES_REQUESTED** [review]

File: bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java:66]: Yes. This is console commant, it speaks to a user through osgi console.

File: bundles/org.eclipse.passage.lbc.jetty/config/log4j2.xml (hunk 3 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lbc.jetty/config/log4j2.xml]: reverted

**Review by ruspl-afed — APPROVED** [review]

**Pull request merged**